modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -601,6 +603,20 @@ public MvPartitionStorage getMvStorage() {
         return minActiveTxBeginTime0;
     }
 
+    /**
+     * Returns minimum starting time among all active RW transactions in the cluster,
+     * or {@code null} if the value has not yet been set.
+     */
+    public @Nullable Long snapshottedMinActiveTxBeginTime() {
```

Comment:
Not sure but may be it worth somehow note the difference between `snapshottedMinActiveTxBeginTime` and `minimumActiveTxBeginTime` in javadocs.
Currently javadocs are identical :thinking: 

Also I suggest to move non-test method upper then methods marked TestOnly.

Reply:
Thanks. Fixed.